### PR TITLE
DS-68: add package Eslint for TailwindCSS (eslint-tailwind-config)

### PR DESCRIPTION
**Pull request by letigarciacom**
###  ℹ️ About

Create the package `eslint-tailwind-config` based on [eslint-plugin-tailwindcss](https://github.com/francoismassart/eslint-plugin-tailwindcss)

- - -

- [x] Has 📖 documentation

- - -
[🎫 Related Ticket DS-68](https://lovetoknow.atlassian.net/browse/DS-68) 
- - -

### CODE REVIEW

File: packages/eslint-tailwind-config/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: ['tailwindcss/recommended'],
+  plugins: ['tailwindcss'],
+  // add the custom rules here
+  rules: {
+    'tailwindcss/classnames-order': 'warn',
+    'tailwindcss/no-custom-classname': 'warn',
```
Comment (paulmelero) [inline packages/eslint-tailwind-config/index.js]: we might want to disable `no-custom-classname` completely. We rely on custom classes heavily...

**Review by paulmelero — APPROVED** [review]
I wasn't expecting this to be a new package, but that's fine! We need to include it by default on the nuxt eslint config too: https://github.com/LoveToKnow/lint/blob/main/packages/eslint-config/nuxt.js

**Review by achimoraites — APPROVED** [review]
🥇  LGTM !!!
Great work @leticiagarcialtk 👏  !!!

**Comment by letigarciacom** [comment]
> I wasn't expecting this to be a new package, but that's fine! We need to include it by default on the nuxt eslint config too: https://github.com/LoveToKnow/lint/blob/main/packages/eslint-config/nuxt.js

Oh, maybe I misunderstood the ticket. I can added in the same eslint package, if it is easier to use in the projects.

**Comment by paulmelero** [comment]
It's ok. We can have it as a separate package. But we're always going to use it with the other eslint configs. I think we want to add it to the eslint readme's:

https://github.com/LoveToKnow/lint/tree/main/packages/eslint-config#installation

The same way we're installing `@lovetoknow/prettier-config` with `@lovetoknow/eslint-config`, in the [Vue](https://github.com/LoveToKnow/lint/tree/main/packages/eslint-config#vue-config) or [Nuxt](https://github.com/LoveToKnow/lint/tree/main/packages/eslint-config#nuxt-config) sections, we could add this one in the installation steps and in the configuration instructions. But we can do it after closing this PR.

**Review by paulmelero — COMMENTED** [review]
Awesome, Leticia! 👏🏽

**Pull request merged**